pkg/cmd/server/start/master_args.go

```diff
@@ -297,7 +297,7 @@ func (args MasterArgs) GetServerCertHostnames() (util.StringSet, error) {
 		return nil, err
 	}
 
-	allHostnames := util.NewStringSet("localhost", "127.0.0.1", masterAddr.Host, masterPublicAddr.Host, kubePublicAddr.Host, assetPublicAddr.Host)
```

Comment:
works for me. @smarterclayton should we also "openshift.default.local", or wait until we're creating that service?